pom.xml

```diff
@@ -79,6 +79,11 @@
         <version>${felix-fileinstall.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>org.springframework.security</groupId>
+        <artifactId>spring-security-core</artifactId>
```

Comment:
Why the new dependency?

Reply:
It was because I added an exclusion to the pentaho-platform-core that was bringing the whole internet with it. However I think I changed that because it was mostly test issues. I'll double check if its still needed.